### PR TITLE
Set default NB/SB election timer to 16s

### PR DESCRIPTION
**Pull request by mkalcok**
Note: This change will affect only new deployments.

### CODE REVIEW

**Comment by fnordahl** [comment]
@mkalcok I tested a version of this rebased onto #83, and tests are now passing, so I'd love to get an updated version!

**Comment by mkalcok** [comment]
rebasing now

File: snapcraft/ovn-central.env [hunk 1]
```diff
@@ -10,3 +10,6 @@ export OVN_RUNDIR="${SNAP_COMMON}/run/ovn"
 export OVN_PKGDATADIR="${SNAP}/share/ovn"
 export OVN_SYSCONFDIR="${SNAP}/etc"
 export OVN_PKIDIR="${SNAP_COMMON}/data/pki"
+
+# Set more lenient election timer for NB/SB clusters (16s)
+export ELECTION_TIMER=16000
```
Comment (fnordahl) [inline snapcraft/ovn-central.env:15]: nit: newline at end of file

**Review by fnordahl — APPROVED** [review]
LGTM, thanks!

File: snapcraft/ovn-central.env (hunk 1 above)
Comment (mkalcok) [reply snapcraft/ovn-central.env:15]: :facepalm:

**Comment by tomponline** [comment]
@mkalcok can you let us know when this lands in latest/stable please?

**Pull request merged**